qa/backwards/shared/src/test/java/org/elasticsearch/test/ESBackcompatTestCase.java

```diff
@@ -189,7 +189,7 @@ protected TestCluster buildTestCluster(Scope scope, long seed) throws IOExceptio
         TestCluster cluster = super.buildTestCluster(scope, seed);
         logger.info("Build internal cluster at {}", new Object[] {cluster.httpAddresses()});
         ExternalNodeServiceClient nodeServiceClient = new ExternalNodeServiceClient();
-        ExternalNode externalNode = new ExternalNode(backwardsCompatibilityVersion(), randomLong(), nodeServiceClient, new NodeConfigurationSource() {
+        ExternalNode externalNode = new ExternalNode(backwardsCompatibilityVersion().toString(), randomLong(), nodeServiceClient, new NodeConfigurationSource() {
```

Comment:
Would it make sense to change the signature of `ExternalNode` too, so it takes a `Version` parameter instead of a `String`? This would ensure at compile time that we pass really just a version instead of an arbitrary string.